frontend/src/store/index.js

```diff
@@ -1094,17 +1099,19 @@ const actions = {
         dispatch('setError', err)
       })
   },
-  setShootListFilters ({ dispatch, commit }, value) {
-    return dispatch('shoots/setShootListFilters', value)
-      .catch(err => {
-        dispatch('setError', err)
-      })
+  async setShootListFilters ({ dispatch, getters }, value) {
+    try {
+      await dispatch('shoots/setShootListFilters', value)
```

Comment:
I know it is no part of this PR but we don't need an action here. We could directly call the mutation in the shoot module.
```
  commit('shoots/SET_SHOOT_LIST_FILTERS', value)
```

Reply:
`shoots/SET_SHOOT_LIST_FILTERS` is working with the rootState.
`modules/shoots.js`:
```js
  SET_SHOOT_LIST_FILTERS (state, { rootState, value }) {
    state.shootListFilters = value
    setFilteredAndSortedItems(state, rootState)
  },
```

 Not sure if I may pass state as rootState like e.g.
```js
commit('shoots/SET_SHOOT_LIST_FILTERS', { rootState: state, value })
```

Reply:
You are right. We need at least the action on the root module. As I said, it not part of this PR. We will refactor that anyway and may get ride of the rootState at all. You can already remove the actions in the shoots module or keep them.

Reply:
as discussed offline we will address this in the upcoming, planned refactoring